examples/src/main.rs

```diff
@@ -29,38 +29,57 @@ fn main() {
-        ExampleType::Merkle { tree_depth } => merkle::get_example(options, tree_depth),
-        #[cfg(feature = "std")]
-        ExampleType::LamportA { num_signatures } => {
-            lamport::aggregate::get_example(options, num_signatures)
-        }
-        #[cfg(feature = "std")]
-        ExampleType::LamportT { num_signers } => {
-            lamport::threshold::get_example(options, num_signers)
-        }
-    };
+    let example =
+        match options.example {
+            ExampleType::Fib { sequence_length } => {
+                fibonacci::fib2::get_example(&options, sequence_length)
+                    .expect("The example failed to initialize.")
+            }
+            ExampleType::Fib8 { sequence_length } => {
+                fibonacci::fib8::get_example(&options, sequence_length)
+                    .expect("The example failed to initialize.")
+            }
+            ExampleType::Mulfib { sequence_length } => {
+                fibonacci::mulfib2::get_example(&options, sequence_length)
+                    .expect("The example failed to initialize.")
+            }
+            ExampleType::Mulfib8 { sequence_length } => {
+                fibonacci::mulfib8::get_example(&options, sequence_length)
+                    .expect("The example failed to initialize.")
+            }
+            ExampleType::FibSmall { sequence_length } => {
+                fibonacci::fib_small::get_example(&options, sequence_length)
+                    .expect("The example failed to initialize.")
+            }
+            ExampleType::Vdf { num_steps } => vdf::regular::get_example(&options, num_steps)
+                .expect("The example failed to initialize."),
+            ExampleType::VdfExempt { num_steps } => vdf::exempt::get_example(&options, num_steps)
+                .expect("The example failed to initialize."),
+            ExampleType::Rescue { chain_length } => rescue::get_example(&options, chain_length)
+                .expect("The example failed to initialize."),
+            #[cfg(feature = "std")]
+            ExampleType::RescueRaps { chain_length } => {
+                rescue_raps::get_example(&options, chain_length)
+                    .expect("The example failed to initialize.")
+            }
+            #[cfg(feature = "std")]
+            ExampleType::Merkle { tree_depth } => merkle::get_example(&options, tree_depth)
+                .expect("The example failed to initialize."),
+            #[cfg(feature = "std")]
+            ExampleType::LamportA { num_signatures } => {
+                lamport::aggregate::get_example(&options, num_signatures)
+                    .expect("The example failed to initialize.")
+            }
+            #[cfg(feature = "std")]
+            ExampleType::LamportT { num_signers } => {
+                lamport::threshold::get_example(&options, num_signers)
+                    .expect("The example failed to initialize.")
+            }
+        };
```

Comment:
Is it possible to put a single `.expect()` here (right after `}`) - or does this not work?

Reply:
It's working, thanks! Please see [fa8e7c4](https://github.com/novifinancial/winterfell/pull/111/commits/fa8e7c408cd9ab80ae3367d8c68b87c9e2231ce1).